extras/rya.export/export.api/pom.xml

```diff
@@ -98,6 +98,24 @@ under the License.
                 </configuration>
             </plugin>
 
+            <plugin>
```

Comment:
Plugin execution is unnecessary if code is stored in ${project.build.directory}/generated-sources.